### ISSUE TITLE
[TDS] Create 'On Time' Performance Measure Report for TIA

### ISSUE THREAD

**Issue by ChrispinP** [cityofaustin/atd-data-tech#5893]
**New On Time Report**

- [x] Custom Filters
- [x] Group by 'Cycle #'
- [x] Calculate by 'On Time' count
- [x] Use 'Days Idle' (Submittal Date to Begin Review Date)
- [x] Count field 'Number of On Time Submissions' (Request object). Filter by Days Idle value.
- [x] % Equation[(# of On Time Scope Submissions + # of On Time Submissions) / (# of Scope Submissions + # of Submissions)]

**Comment by ChrispinP** [comment]
Built 2 area reports for Scope Submissions and Submissions for this metric. Added metric statistics to case management page.

Will need to determine if this meets need and if any further adjustments are needed.

**Comment by ChrispinP** [comment]
Updated Submission reports to be on their own reporting page to create clarity. I also provided legends next to these performance metrics so they are easier to understand. 

![image](https://user-images.githubusercontent.com/67915167/117078203-0a0a0980-acff-11eb-817f-0c42d24ce56e.png)

I updated the equations themselves so they accurately reflect the raw day value for average submission cycle times. In addition, I modified the area charts after a better understanding of them and included the "Late" or not on time value so combined the On Time and Late values will = 100%

![image](https://user-images.githubusercontent.com/67915167/117078419-738a1800-acff-11eb-800c-633b8fdb8c05.png)
